external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSourceSuite.scala

```diff
@@ -303,94 +302,75 @@ class KafkaMicroBatchSourceSuite extends KafkaSourceSuiteBase {
     )
   }
 
-  testWithUninterruptibleThread(
```

Comment:
I think this test is superfluous and does not test anything useful. As with the other modified tests, "simulating" an implementation is a BAD test, and in this particular case it is attempting to simulate the 2.1.0 log, which is not necessary any more.

Reply:
Added